### DISCUSSION TITLE
What's the difference between MCP and vector database?

### DISCUSSION THREAD

**Discussion by iamysfy** [modelcontextprotocol/modelcontextprotocol#1242] [Q&A - General] [answered]
it's been a while and I can't figure it out

**Comment by andrewlayer** [comment, marked as answer]
You can think of MCP as an agreed upon language that two systems agree to communicate to one and other in.  For example, I am communicating with you in english, english is the "protocol" you and I both agree to communicate in so that we can understand one and other.  

On the other hand, a vector databases try to find similar phrases.  So if you have a vector database populated with the following:

`I like pizza`
`where is toronto`
`when was lincoln born`

And I query the database asking "what is your favorite food" the most similar sentence to that is `I like pizza`.  The two concepts are pretty different. Hopes this helps

**Reply by iamysfy** [reply]
I really apprectiate the time you took to explain, thank you.

well understood.

**Comment by localden** [comment]
We've limited Discussions in this repository to meeting notes and are closing this thread. To continue the conversation, pick a channel from [Contributor Communication](https://modelcontextprotocol.io/community/communication). Thanks for posting.
